airbyte-integrations/connectors-performance/destination-harness/src/main/java/io/airbyte/integrations/destination_performance/Main.java

```diff
@@ -46,6 +51,12 @@ public static void main(final String[] args) {
         dataset = args[1];
         numOfParallelStreams = Integer.parseInt(args[2]);
       }
+      case 4 -> {
```

Comment:
nit: with more arguments we might want at some point to switch to a more robust parsing by arg=value rather than number or order.